lib/ansible/modules/database/postgresql/postgresql_sequence.py

```diff
@@ -0,0 +1,748 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Tobias Birkefeld (@tcraxs) <t@craxs.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: postgresql_sequence
+short_description: Create, drop, or alter a PostgreSQL sequence
+description:
+- Allows to create, drop or change the definition of a sequence generator
+  U(https://www.postgresql.org/docs/current/sql-createsequence.html).
+version_added: '2.8'
+options:
+  sequence:
+    description:
+    - The name (optionally schema-qualified) of the sequence.
+    required: true
+    type: str
+    aliases:
+    - name
+  state:
+    description:
+    - The sequence state.
+    - I(state=absent) is mutually exclusive with I(data_type), I(increment), I(minvalue), I(maxvalue),
+      I(start), I(cache), I(cycle), I(rename_to) , I(newschema) and I(owner).
+    default: present
+    choices: [ absent, present ]
+    type: str
+  data_type:
+    description:
+    - Specifies the data type of the sequence. Valid types are smallint, integer, and bigint. bigint is the default.
+      The data type determines the default minimum and maximum values of the sequence.
+    - Supported from PostgreSQL 10.
+    required: false
+    choices: [ smallint, integer, bigint ]
```

Comment:
there is a recommendation to put params alphabetically

Reply:
thx, I changed it